src/set_theory/surreal.lean

```diff
@@ -146,12 +146,12 @@ begin
    ... ≃ xr × yl ⊕ xl × yr : equiv.sum_comm _ _
    ... ≃ yl × xr ⊕ yr × xl : equiv.sum_congr (equiv.prod_comm _ _) (equiv.prod_comm _ _)
    ... ≃ (y * x).right_moves : by refl,
-  { rintro (⟨i, j⟩ | ⟨i, j⟩),
+  { rintro (⟨i, j⟩|⟨i, j⟩),
```

Comment:
I think the way it was before is the preferred style for `rintro`, but I could be mistaken.

Reply:
I'll revert it, thanks.